### PR TITLE
Do not log all plugin files per each product test setup

### PR DESCRIPTION
**Pull request by kokosing**
Do not log all plugin files per each product test setup

### CODE REVIEW

**Comment by kokosing** [comment]
It is all about the scrolling log files.

**Comment by kokosing** [comment]
In my case there were over 4500 of log lines from io.prestosql.server.PluginManager. Notice that is logged per each presto docker container.

File: presto-product-tests/conf/presto/etc/log.properties [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #
 
 io.prestosql=INFO
+io.prestosql.server.PluginManager=WARN
```
Comment (findepi) [inline presto-product-tests/conf/presto/etc/log.properties]: i agree this is too verbose.
I am concern WARN may be too little.

maybe we got `PluginManager` log levels incorrectly in https://github.com/prestosql/presto/pull/1311?

**Comment by kokosing** [comment]
I reverted the #1311

File: presto-product-tests/conf/presto/etc/log.properties (hunk 1 above)
Comment (electrum) [reply presto-product-tests/conf/presto/etc/log.properties]: I agree, it seems too verbose to me.

**Pull request merged**